k8s/postgresql/manifest/manifests.yaml.template

```diff
@@ -0,0 +1,93 @@
+kind: ServiceAccount
```

Comment:
Consider to move this service account into the `schema.yaml` as described [here](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/tree/b83fb9c7e2ebbab7520fa21498138aa1a9caf958#type-service_account).

See [Grafana's schema.yaml](https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/grafana/schema.yaml#L27-L37).